.ci/containers/terraform-tester/test_terraform.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+REPO=$1
+REFERENCE=$2
+SCRATCH_OWNER=modular-magician
+if [ "$REPO" == "ga" ]; then
+    GH_REPO=terraform-provider-google
+    LOCAL_PATH=$GOPATH/src/github.com/terraform-providers/terraform-provider-google
+elif [ "$REPO" == "beta" ]; then
+    GH_REPO=terraform-provider-google-beta
+    LOCAL_PATH=$GOPATH/src/github.com/terraform-providers/terraform-provider-google-beta
+else
+    echo "no repo, dying."
+    exit 1
+fi
+
+SCRATCH_PATH=https://modular-magician:$GITHUB_TOKEN@github.com/$SCRATCH_OWNER/$GH_REPO
```

Comment:
any reason to hardcode "modular-magician" in the first half of the string but not in the second half?